src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+// Action Types
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// Initial State
+const initialState = [];
+
+// Action Creators
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  book,
+});
+
+export const removeBook = (book) => ({
+  type: REMOVE_BOOK,
+  book,
+});
+
+// Reducer
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.book];
+    case REMOVE_BOOK:
+      return state.filter((b) => b !== action.book);
```

Comment:
- It is advisable to filter by the id and not by the book object. I suggest filtering the book this way return `state.filter((b) => b.id !== action.book.id)`. Please fix this 🙏

Reply:
@mckent05 Done. Thanks a lot!